deploy/charts/external-secrets/templates/deployment.yaml

```diff
@@ -104,4 +105,117 @@ spec:
       {{- if .Values.priorityClassName }}
       priorityClassName: {{ .Values.priorityClassName }}
       {{- end }}
+{{- else }}
+{{- $values := .Values }}
+{{- $chart := .Chart }}
+{{- $release := .Release }}
+{{- range $index, $namespace := .Values.scopedNamespaces }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: "{{ include "external-secrets.fullname" $ }}-{{ $namespace }}"
+  namespace: {{ $release.Namespace | quote }}
```

Comment:
This is going to install all the operators in the same namespace, bound to the same service account. If we are going to go to this route (create multiple operators per namespace), we should at least create all of its underlying resources as well.